src/solvers/refinement/string_refinement.h

```diff
@@ -171,6 +171,9 @@ class string_refinementt: public bv_refinementt
   std::string string_of_array(const array_exprt &arr);
 };
 
+#define string_invariantt(reason) \
```

Comment:
Shouldn't we be using uppercase for macros? (I don't really know what the convention is.)